downstream/modules/platform/proc-gw-define-rules-triggers.adoc

```diff
@@ -18,32 +18,22 @@ Always:: The trigger should always be fired.
+* *Groups:* This is a list of one or more groups coming from the authentication system that the user must be a member of. See the Operation field to determine the behavior of the trigger if more than one group is specified in the trigger.
+* *Attribute:* The map is true or false based on a users attributes coming from the source system.When defining an attribute trigger, the authentication mapping expands to include the following selections:
+* *Operation:* This field includes conditional settings that trigger the handling of the rule based on the specified *Attribute* criteria. In version 2.5.0 this field needs to be set but is ignored. In the future, attribute based maps will allow for the processing of multiple attributes. *NOTE:* If you would like to experiment with multiple attribute maps you can do that through the API but the UI form will remove multi-attribute maps if the authenticator is saved through the UI.
+* *Attribute:* The name of the attribute coming from the source system this trigger will be evaluated against. For example, if you wanted the trigger to fire based on the user's last name and the last name field in the source system was called users_last_name you would enter the value ‘users_last_name’ in this field.
+* *Comparison:* Tells the trigger how to evaluate the value of the users. *Attribute* in the source system compared to the *Value* specified on the trigger. Available options are: *contains*, *matches*, *ends with*, *in*, or *equals*. Below is a breakdown of each comparison type:
++
+** *contains*: The specified character sequence in *Value* is contained within the attributes value returned from the source. For example, given an attribute value of ‘John’ from the source the contains *Comparison* would set the trigger to *True* if the trigger *Value* was set to ‘Jo’ and False if the trigger *Value* was ‘Joy’.
+** *matches*: The *Value* on the trigger is treated as a python regular expression and does an link::https://docs.python.org/3/library/re.html#re.match[re.match] (with case ignore on) between the specified *Value* and the value returned from the source system. For example, if the trigger's *Value* was ‘Jo’ the trigger would return *True* if the value from the source was John or Joanne or any other value which matched the regular expression ‘Jo’. The trigger would return False if the sources value for the attribute was ‘Dan’ because ‘Dan’ does not match the regular expression ‘Jo’. 
```

Comment:
link formatting here has a typo--two colons preventing it from rendering correctly. And should it be "does *a* re.match?

Reply:
Fixed. It's a regular expression match which is called re.match.